src/routes/wish-statistics/overview/+page.svelte

```diff
@@ -24,10 +24,10 @@
 	import i18n from '$lib/services/i18n';
 	import { userProfile } from '$lib/store/user_profile';
 	import { applicationState } from '$lib/store/application_state';
+	import { get } from 'svelte/store';
 
 	let wishData: IMappedWishes = {};
-	const wishes: IWishes | undefined = $userProfile.wishes;
-
+	const wishes: IWishes | undefined = get(userProfile).wishes;
```

Comment:
I need to refactor this. Coming from react i though that the value of wishes would update if i used $<store> and cause a rerender. But it seems that Svelte only rerenders if a $<store> var changes in template instead of script

Reply:
well i'll let you do it on another PR then